.gitignore

```diff
@@ -47,6 +48,7 @@ fastlane/README.md
 fastlane/test_output/
 Preview.html
 build/
+screenshots/
```

Comment:
Fastlane dumps this in the root folder. I left it there for now, but I think it shall go in the `fastlane/` folder, as per what WooCommerce iOS does.

Reply:
Yep definitely should – can we avoid the `.gitignore` change since we'll just undo it later, and for now you could use git exclude on it if it's in the way? :)